### ISSUE TITLE
[Training] Clarification on training sentence-level VSR

### ISSUE THREAD

**Issue by EidanErlich** [KAIST-AILab/SyncVSR#3]
Great work on this work! 

I’m currently working with sentence-level datasets (LRS2, LRS3, Vox2) and I’m a bit lost on how to run the repo with these datasets. Could you provide additional instructions or guidance on how to run sentence-level Visual Speech Recognition (VSR) using this setup?

Thanks in advance!

**Comment by snoop2head** [comment]
@EidanErlich 

Thank you for your encouragement! 
We've updated the code for the sentence level VSR experimentation code at the repo.
Please refer to the implementation attached below.

https://github.com/KAIST-AILab/SyncVSR/blob/4f51376eb023d44c67818c063235c30192d212d1/LRS/video/espnet/nets/pytorch_backend/e2e_asr_transformer.py#L185-L227

**Comment by snoop2head** [comment]
@EidanErlich 

I will close the issue for now.
In order to align audio with the video, please refer to the attached the code above and [newly uploaded audio tokens](https://github.com/KAIST-AILab/SyncVSR/releases/tag/weight-audio-v1) for your reference.

If you have any other inquiries, please re-open the issue or submit a new one!

**Comment by snoop2head** [comment]
@EidanErlich 

Kindly refer to the diagram below for sentence-level VSR. As mentioned in the code attached in https://github.com/KAIST-AILab/SyncVSR/issues/3#issuecomment-2389266946 above, the encoder-decoder model is trained using a composite loss function that includes LM Loss, CTC Loss, and our Audio Reconstruction Loss. In our framework, audio-video synchronization is achieved by aligning each video frame with its corresponding set of audio tokens.

<img width="450" alt="image" src="https://github.com/KAIST-AILab/SyncVSR/blob/main/assets/Sentence_Level_VSR.png?raw=true">
